js/compiler/compiler.go

```diff
@@ -124,7 +132,28 @@ func (c *Compiler) Transform(src, filename string, inputSrcMap []byte) (code str
 		return
 	}
 
-	code, srcMap, err = c.babel.transformImpl(c.logger, src, filename, c.Options.SourceMapLoader != nil, inputSrcMap)
+	sourceMapEnabled := c.Options.SourceMapLoader != nil
+	maxSrcLenForBabelSourceMapOnce.Do(func() {
+		// TODO: drop this code and everything it's connected to when babel is dropped
```

Comment:
Do we have an issue to link?

Reply:
https://github.com/grafana/k6/issues/2296#issuecomment-993356717 maybe? :thinking:

Reply:
mmm...if we don't have one yet, I'm for creating one dedicated with the steps we plan for dropping babel. For example, it would be useful to have somewhere a link to https://github.com/dop251/goja/issues/348 and this potential new issue could be a good candidate, IMHO.